PWGHF/TableProducer/candidateCreatorToXiPi.cxx

```diff
@@ -233,6 +237,11 @@ struct HfCandidateCreatorToXiPi {
         auto groupedTrackIndices = trackIndices.sliceBy(trackIndicesPerCollision, thisCollId);
         for (const auto& trackIndexPion : groupedTrackIndices) {
 
+          // use bachelor selections from HfTrackIndexSkimCreatorTagSelTracks --> bit =2 is CandidateType::CandV0bachelor
+          if (!TESTBIT(trackIndexPion.isSelProng(), 2)) {
+            continue;
+          }
```

Comment:
Why don'y ypou use `CandidateType::CandV0bachelor` in the bit check?

Reply:
Because the enum for CandidateType is in the trackIndexSkimsCreator and not in the header where reconstruction tables are declared, so it is not recognized

Reply:
These lines are used in the process function that runs on "normal" data, not on the derived data